app/components/FileOperationListItem.js

```diff
@@ -0,0 +1,158 @@
+// @flow
+import { LinkIcon } from "outline-icons";
+import * as React from "react";
+import { useTranslation } from "react-i18next";
+import { Link } from "react-router-dom";
+import styled, { css } from "styled-components";
+import breakpoint from "styled-components-breakpoint";
+import FileOperation from "models/FileOperation";
+import Avatar from "components/Avatar";
+import Badge from "components/Badge";
+import Button from "components/Button";
+import EventBoundary from "components/EventBoundary";
+import Flex from "components/Flex";
+import Time from "components/Time";
+import useCurrentUser from "hooks/useCurrentUser";
+import useMobile from "hooks/useMobile";
+import useQuery from "hooks/useQuery";
+
+type FileOperationListItemProps = {
+  item: FileOperation,
+};
+
+const FileOperationListItem = ({ item }: FileOperationListItemProps) => {
+  const user = useCurrentUser();
+  const { t } = useTranslation();
+  const isMobile = useMobile();
+  const params = useQuery();
+  const key = params.get("key") || "";
+
+  return (
+    <ImportExportListItemWrapper highlight={key === item.id}>
+      <Flex align="center" column>
+        <AvatarWrapper src={item.user.avatarUrl} size={32} />
```

Comment:
A big avatar seems overkill for this type of item, we could go with an icon or maybe nothing at all is needed.

Sidenote: I know there was no design given 😉